src/components/Article/Byline/Byline.styled.js

```diff
@@ -67,20 +65,23 @@ Content.propTypes = {
   children: PropTypes.oneOfType([PropTypes.array, PropTypes.object])
 }
 
-export const DateLine = ({ children, custom }) => 
+export const DateLine = ({ children, mb }) => 
   <Block 
-    custom={custom}
-    mb='2'
-    md={{
-      mb: '0px'
+    mb={mb[0]}
+    lg={{
+      mb: mb[1]
     }}
     width='calc(100% - 76px)'>
     {children}
   </Block>
 
+DateLine.defaultProps = {
+  mb: ['4', '4'],
```

Comment:
Acho que aqui tem que ser
```suggestion
  mb: [2, '0px'],
```

Caso contrário vamos propagar o novo valor de '4' para todos os fronts

Reply:
Não vejo porque aplicar isso. Todos os fronts vão usar a mesma estrutura do Byline, como comentei no conversation anterior.